install.js

```diff
@@ -25,128 +26,159 @@ function onerror(err) {
  * @param cb Callback
  */
 function install(runtime, abi, platform, arch, cb) {
-  const essential = runtime + '-v' + abi + '-' + platform + '-' + arch;
-  const pkgVersion = pkg.version;
-  const currentPlatform = pkg.name + '-v' + pkgVersion + '-' + essential;
-
-  console.log('Downloading prebuild for platform:', currentPlatform);
-  let downloadUrl = 'https://github.com/wilix-team/iohook/releases/download/v' + pkgVersion + '/' + currentPlatform + '.tar.gz';
-
-  let nuggetOpts = {
-    dir: os.tmpdir(),
-    target: 'prebuild.tar.gz',
-    strictSSL: true
-  };
-
-  let npmrc = {};
-
-  try {
-    rc('npm', npmrc);
-  } catch (error) {
-    console.warn('Error reading npm configuration: ' + error.message);
-  }
-
-  if (npmrc && npmrc.proxy) {
-    nuggetOpts.proxy = npmrc.proxy;
-  }
-
-  if (npmrc && npmrc['https-proxy']) {
-    nuggetOpts.proxy = npmrc['https-proxy'];
-  }
-
-  if (npmrc && npmrc['strict-ssl'] === false) {
-    nuggetOpts.strictSSL = false;
-  }
-
-  nugget(downloadUrl, nuggetOpts, function(errors) {
-    if (errors) {
-      const error = errors[0];
-
-      if (error.message.indexOf('404') === -1) {
-        onerror(error);
-      } else {
-        console.error('Prebuild for current platform (' + currentPlatform + ') not found!');
-        console.error('Try to build for your platform manually:');
-        console.error('# cd node_modules/iohook;');
-        console.error('# npm run build');
-        console.error('');
-      }
+    const essential = runtime + "-v" + abi + "-" + platform + "-" + arch;
+    const pkgVersion = pkg.version;
+    const homepage = github(pkg);
```

Comment:
Ummmm... I don't think this is a good way to do it. Also, the project hasn't been maintained since 2012 and the github APIs change all the time so this doesn't really belong in a build script. Neither does the hardcoded one, but this solution doesn't make things better.

Reply:
This is the exact technique used when [uploading](https://github.com/prebuild/prebuild/blob/master/upload.js#L12) artifacts, I just reused it for downloading

Reply:
In that case keep it as is.